CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ Mapbox welcomes participation and contributions from everyone.
 #### Features
 #### Bug fixes and improvements
 
+## Mapbox Navigation SDK 2.7.3 - 27 January, 2023
+### Changelog
+[Changes between v2.7.2 and v2.7.3](https://github.com/mapbox/mapbox-navigation-android/compare/v2.7.2...v2.7.3)
```

Comment:
IMO that's weird releasing a patch without any changes. We could at least add something about "stabilising and bugfixing". Or we could be more precise. @Guardiola31337 
Just as a suggestion for future patches.

Reply:
Yeah, totally agree - I was thinking the same after release kicked off. Let me fix the `CHANGELOG` 👍

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/6916